### ISSUE TITLE
Security.MD template

### ISSUE THREAD

**Issue by Jonasvdbo** [ospo-nl/kennisbank#11]
Hierin is de Responsible Disclosure opgenomen van het NCSC ?
Of een variant?

**Comment by RietveldBLD** [comment]
Ja, het is inderdaad de RD van NCSC. Ik heb de Security.MD toegevoegd aan /Docs

**Comment by RietveldBLD** [comment]
> Ik zou graag zien dat we iets duidelijkere benaming geven aan issues, bestanden en PRs. SECURITY.md ... zegt mij niet zoveel (of misschien té veel 😄)
> 
> Daarnaast lijkt dit een kopie te zijn van een document ergens anders. Klopt dat? Moeten we dan niet vanuit hier naar dat document refereren en het niet kopiëren? 🤔

Ik heb heel vriendelijk een pull-request vanuit het issue op het board gemaakt. Maar eens, dan moet het issue op het projectboard wel helderder zijn 👍 

Hmm, staat in deze geen referentie? Dan zorg ik dat ik die nog ga toevoegen

**Comment by marcvanandel** [comment]
No worries! 't Is een vriendelijke aanmoediging om vooral door te gaan ... en bedoeld als constructieve feedback 😊
